test/support/controller_test_helpers.rb

```diff
@@ -20,18 +16,22 @@ def should_be_a_public_facing_controller
       end
     end
 
-    def should_require_fatality_handling_permission_to_access(edition_type, *actions)
-      test "requires the ability to handle fatalities to access" do
-        edition = create(edition_type) # rubocop:disable Rails/SaveBang
-        login_as :writer
-        actions.each do |action|
-          get action, params: { id: edition.id }
-          assert_response 403
+    def should_render_bootstrap_implementation_with_preview_next_release
+      test "should render the admin layout when the user has 'Preview next release' permission" do
+        return unless @controller.class.private_method_defined?(:get_layout)
+
+        user = login_as(:writer)
+        user.permissions << "Preview next release"
+
+        # Get all non-inherited controller actions and test they use the admin layout.
+        @controller.class.instance_methods(false).map(&:to_s).reject { |action| LAYOUT_ALWAYS_DESIGN_SYSTEM.include?(action) }.each do |action|
+          @controller.action_name = action
+          assert_equal "admin", @controller.send(:get_layout), "#{@controller.class}##{action} is not rendering the admin layout"
         end
       end
```

Comment:
Very clever test 🤓 looks like it should be handy to catch any mistakes while we're still in this transitionary period.